### PR TITLE
add link to python

### PR DESCRIPTION
**Pull request by zoepan00**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -77,3 +77,7 @@ This is a basic example which shows you how to solve a common problem:
 ## basic example code
 ## TODO
 ```
+
+## Package in python
+
+We have package with same functionalities in python `pyimager`. See [here](https://github.com/UBC-MDS/rimager)
```
Comment (Keanna-K) [inline README.Rmd]:
```suggestion
We have a package with the same functionalities in Python : `pyimager`. See [here](https://github.com/UBC-MDS/pyimager)  
```

File: README.md [hunk 2]
```diff
@@ -88,3 +88,8 @@ This is a basic example which shows you how to solve a common problem:
 ## basic example code
 ## TODO
 ```
+
+## Package in python
+
+We have package with same functionalities in python `pyimager`. See
+[here](https://github.com/UBC-MDS/rimager)
```
Comment (Keanna-K) [inline README.md]:
```suggestion
[here.](https://github.com/UBC-MDS/pyimager)
```

File: README.md [hunk 3]
```diff
@@ -88,3 +88,8 @@ This is a basic example which shows you how to solve a common problem:
 ## basic example code
 ## TODO
 ```
+
+## Package in python
+
+We have package with same functionalities in python `pyimager`. See
```
Comment (Keanna-K) [inline README.md]:
```suggestion
We have a package with the same functionalities in Python : `pyimager`. See
```

**Review by Keanna-K — CHANGES_REQUESTED** [review]
The link was to our R package, so I adjusted it to reference the python package. I added some grammar suggestions. Please make these changes before we pull this edit.

File: README.Rmd [hunk 4]
```diff
@@ -80,4 +80,4 @@ This is a basic example which shows you how to solve a common problem:
 
 ## Package in python
 
-We have package with same functionalities in python `pyimager`. See [here](https://github.com/UBC-MDS/rimager)  
+We have the package with same functionalities in python: `pyimager`. See [here](https://github.com/UBC-MDS/rimager)
```
Comment (Keanna-K) [inline README.Rmd]:
```suggestion
We have a package with the same functionalities in Python: `pyimager`. See [here.](https://github.com/UBC-MDS/pyimager)  
```

File: README.md [hunk 5]
```diff
@@ -91,5 +91,5 @@ This is a basic example which shows you how to solve a common problem:
 
 ## Package in python
 
-We have package with same functionalities in python `pyimager`. See
-[here](https://github.com/UBC-MDS/rimager)
+We have the package with same functionalities in python: `pyimager`. See
```
Comment (Keanna-K) [inline README.md]:
```suggestion
We have a package with the same functionalities in Python: `pyimager`. See
```

**Review by Keanna-K — CHANGES_REQUESTED** [review]
One of the links was still incorrect, please fix this before we merge

**Review by clsu22 — APPROVED** [review]

**Review by Keanna-K — APPROVED** [review]

**Pull request merged**